### ISSUE TITLE
Unread color text "Login to Enroll" in the block "Get Started" on the page "https://www.qax-camp.com.ua/courses/qa-basic-programs/"

### ISSUE THREAD

**Issue by BatalovaOlha** [scholokov/qax-portal-2#367]
**STR**
1. Open https://www.qax-camp.com.ua/courses/qa-basic-programs/ 
2. Pay attention to the block which is placed under the title of the page "Програми для QA" part "Get Started"

**EXPECTED RESULT**
In the block "Get Started" shown white text (#FFFFFF) "Login to Enroll" on a white background (#E9E8EA)

**ACTUAL RESULT**
In the block "Get Started" the text color "Login to Enroll" is contrasting the background

![F0po6KykE5](https://user-images.githubusercontent.com/109384052/209957160-77672907-2e48-4fa7-9432-b55849c17cc7.png)
![NbIr0CZmcI](https://user-images.githubusercontent.com/109384052/209957176-01de7218-6dec-434b-8f3f-93964c819217.png)
![Y2xlGyat7Z](https://user-images.githubusercontent.com/109384052/209957195-425ad493-9ffe-43a4-9309-a07cd8f66587.png)

**Comment by DmytroMarkulych** [comment]
@BatalovaOlha Спочатку має бути Actual Result, а потім Expected Result

**Comment by BatalovaOlha** [comment]
Так, суть була вірною, а назви надрукувала некоректно...(
Змінила

**Comment by DmytroMarkulych** [comment]
@BatalovaOlha так, суть вірна. Я про те, що у нас є кроки для відтворення, відповідно після того як ми відтворюмо дефект по кроках, то отримуємо актуальний результат, а вже після нього очікуваний
